core/src/navigation/LeftNav.html

```diff
@@ -209,6 +221,9 @@
 
 <style type="text/scss">
   $topNavHeight: 48px;
```

Comment:
```suggestion
  $topNavHeight: 48px;
  $side-nav-footer--padding: 13px;
  $side-nav-footer--font-size: 12px;
  $side-nav-footer--line-height: 16px;
```

Let's use some variables. Check my comments below.